packages/merkle-distributor/package.json

```diff
@@ -8,15 +8,13 @@
     "@uma/core": "^2.0.1",
     "chai": "^4.3.0",
     "commander": "^7.1.0",
+    "ethereumjs-util": "^7.0.9",
```

Comment:
Took me a while to figure this out: this package includes the `types` needed for line 3 of src/MerkleTree.ts, not intuitively the `@types/ethereumjs-util` package.

Reply:
Does it only need the types? Are there any runtime deps pulled from it? If it's _only_ types, we could move it to devDependencies.

Reply:
It only needs types i can move to devDeps